### DISCUSSION TITLE
cc.acceptedCategory() not working as expected

### DISCUSSION THREAD

**Discussion by kallefrombosnia** [orestbida/cookieconsent#733] [Q&A]
Hello, suddenly after project update to [Livewire "SPA" navigation](https://livewire.laravel.com/docs/navigate), `CookieConsent` starts to return wrong values on `acceptedCategory()`.

Steps to reproduce:
1. Accept all cookies / or full page refresh (normal HTTP GET)
2. Console -> cc.acceptedCategory('analytical') -> `true`
3. Redirect via `wire:navigate` to any other page
4. Console -> cc.acceptedCategory('analytical') -> `false`

Can confirm that cookie has given category as approved.

This worked fine before this SPA update, but now its creating problems, since GA doesnt get `granted` on specific targeting.
I had it fixed by manualy checking the cookie value and granting approval on new page load, but still `acceptedCategory` returns false when redirected.

I didnt understand if this was for the Github issues so I decided to ask here.

Thank you.

**Comment by orestbida** [comment]
I tried setting up a basic Laravel + LiveWire app and onfigured the plugin to be executed once, in the root layout (with persistent html across pages).

Everything works as expected on my end.

Note that the plugin is executed asynchronously. Make sure you are not calling any method before the plugin is initialized. Does calling `cc.acceptedCategory('analytical')` after 1 second, still give the same wrong result? If so, then there's a configuration issue with LiveWire - cookieconsent.
